tests/appsec/iast/sink/test_header_injection.py

```diff
@@ -3,7 +3,7 @@
 # Copyright 2021 Datadog, Inc.
 
 from utils import context, features, missing_feature
-from ..utils import BaseSinkTest
+from ..utils import BaseSinkTest, BaseTestHeaderInjectionReflectedExclusion
```

Comment:
`BaseTestHeaderInjectionReflectedExclusion` is Header related test, why are you creating the class in `utils` and not here?

Reply:
Fixed :)